packages/core/src/change_detection/differs/iterable_differs.ts

```diff
@@ -113,14 +113,52 @@ export interface IterableChangeRecord<V> {
 }
 
 /**
- * An optional function passed into the `NgForOf` directive that defines how to track
- * changes for items in an iterable.
- * The function takes the iteration index and item ID.
- * When supplied, Angular tracks changes by the return value of the function.
+ * A function optionally passed into the `NgForOf` directive in order to customize how `NgForOf`
```

Comment:
```suggestion
 * A function optionally passed into the `NgForOf` directive to customize how `NgForOf`
```
Style guide preference: If at all possible, don't use in order to; instead, use to.